### PR TITLE
added props showSuffix

### PR DESCRIPTION
**Pull request by RenatoBueno-A**
#### What problem is this solving?

We are migrating a cms store to Vtex IO by inserting the component `quantity-selector` was rendered the unit of measure in the quantity selector, becoming different from the proposed layout

#### How to test it?

access the workspace and see the quantity selector showing the unit of measurement
[Workspace showSuffixTrue](https://showsuffix--tokstokio.myvtex.com/triang-caneca-espresso-80-ml-ouro-preto-hipfix/p)

access the workspace and see the quantity selector without the unit of measure
[Workspace showSuffixFalse](https://showsuffixfalse--tokstokio.myvtex.com/triang-caneca-espresso-80-ml-ouro-preto-hipfix/p)

#### Screenshots or example usage:

```diff
 "product-quantity": {
    "props": {
      "showSuffix": true
    }
  },
```
#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Added prop `showSuffix` for show or hide suffix on `quantity-selector`
```
Comment (kaisermann) [inline CHANGELOG.md:10]:
```suggestion
- Added prop `showSuffix` to show or hide suffix on `quantity-selector`
```

File: docs/README.md [hunk 2]
```diff
@@ -61,6 +62,7 @@ You are now able to use all blocks that are exported by the Product Quantity app
 | Prop name | Type | Description | Default Value |
 | --- | --- | --- | --- |
 | `warningQuantityThreshold` | `number` | Displays the quantity of remaining items in stock if the available quantity is less than or equal to the value given to this property. | `0` (the quantity is not displayed) |
+| `showSuffix` | `boolean` |Whether the unit of measurement should be displayed (`true`) or not (`false`). | `true` |
```
Comment (kaisermann) [inline docs/README.md:65]: What do you think of "showUnit"?

Comment (RenatoBueno-A) [reply docs/README.md:65]: it's ok for me, I'll make this change

**Review by guerreirobeatriz — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Added prop `showUnit` to show or hide suffix on `quantity-selector`
```
Comment (kaisermann) [inline CHANGELOG.md:10]:
```suggestion
- Added prop `showUnit` to show or hide unit on `quantity-selector`
```

**Review by kaisermann — APPROVED** [review]

**Pull request merged**